docs/docusaurus/src/css/custom.scss

```
@@ -299,6 +299,10 @@ table.borderless tr:nth-child(2n) {
   line-height: var(--p-line-height)
 }
 
+div.theme-doc-toc-mobile {
+  display: block;
+}
```

This should only work on mobile, can you try this change?
```suggestion
div.theme-doc-toc-mobile {
  display: none;
  
  @media (max-width: $mobile-breakpoint) {
    display: block;
  }
}
```

---

You are absolutely right! I thought docusaurus applied either that class or `.theme-doc-toc-desktop` depending on the viewport but you are right!